coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20171122-add-secondary-move-jobs-table.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="Alexa Rust" id="20171122-add-secondary-move-jobs-table">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="secondary_move_jobs"/>
+            </not>
+        </preConditions>
+        <createTable tableName="secondary_move_jobs">
+            <column name="dataset_system_id" type="BIGINT">
+                <constraints nullable="false"
+                             foreignKeyName="secondary_move_jobs_dataset_system_id_fkey"
+                             references="dataset_map(system_id)"
+                />
+            </column>
+            <column name="from_store_id" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="move_from_store_complete" type="BOOLEAN" defaultValueBoolean="false">
+                <constraints nullable="false"/>
+            </column>
+            <column name="to_store_id" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="move_to_store_complete" type="BOOLEAN" defaultValueBoolean="false">
```

Comment:
Maybe make this a timestamp "completed at" to let you do a bit of tracking by querying the table?